src/promql/src/extension_plan/empty_metric.rs

```diff
@@ -242,8 +288,34 @@ impl Stream for EmptyMetricStream {
     }
 }
 
+/// Build a schema that only contains **millisecond** timestamp column
+fn build_ts_only_schema(column_name: &str) -> DFSchema {
+    let ts_field = DFField::new(
+        Some(""),
+        column_name,
+        DataType::Timestamp(TimeUnit::Millisecond, None),
+        false,
+    );
+    // safety: should not fail (UT covers this)
```

Comment:
@waynexia just for my understanding what does `UT` stand for?

Reply:
Looks like it means unit test here.